praw/models/comment_forest.py

```diff
@@ -72,6 +72,12 @@ def __len__(self) -> int:
         """Return the number of top-level comments in the forest."""
         return len(self._comments)
 
+    def __eq__(self, other: _CommentForest):
+        """Compare two comment forest instances."""
+        if isinstance(other, CommentForest):
```

Comment:
Same comment that [I made](https://github.com/praw-dev/praw/pull/1369#discussion_r396024824) in #1369: I think the type annotation is probably incorrect.